site/cds_rdm/inspire_harvester/jobs.py

```diff
@@ -116,6 +112,9 @@ def build_task_arguments(
         cls, job_obj, since=None, inspire_id=None, until=None, on=None, **kwargs
     ):
         """Build task arguments."""
+        if isinstance(since, datetime):
+            since = since.date().strftime("%Y-%m-%d")
```

Comment:
Needed because `since` is a datetime when coming from the last successful run: https://github.com/inveniosoftware/invenio-jobs/blob/master/invenio_jobs/jobs.py#L105